DEVELOPER.md

```diff
@@ -0,0 +1,79 @@
+# Developer conventions
+
+## Typescript
+
+### Naming conventions
+
+_**TODO:** Should we use units formats for e.g. number values (`timeSinceYesterdayInMs` | `activatedRundownAtEpochTime`)._
+
+### Logging
+Production log level should be `info`, in order to have a sufficient amount of information when a incident occurs.
+From this decision, it is important that we limit the use of info log statements, such that we don't drown out the important information.
+
+A log entry should at least contain the following:
+```typescript
+interface LogEntry {
+    level: 'error' | 'warn' | 'info' | 'debug' | 'trace',
+    timestamp: number,
+    message: string,
+    tag: string // The owner of the log statement (e.g. RundownService)
+    data?: unknown, // Potential information, e.g. error message including stack trace.
+}
+```
+
+## SASS/SCSS
+
+### Naming conventions
+
+**Variables**
+
+_**TODO:** Find out how to handle global settings such as color palettes, sizings, etc..._
+
+**Attribute order**
+
+When writing scss the suggested attribute order is:
```

Comment:
Should we emphasize that the goal here is to have consistency?

Reply:
Yes. I have updated it to reflect that.